index.html

```diff
@@ -62,6 +62,177 @@ <h1 class="header-title">I’m Abdallah Glad to see you!</h1>
            </li>    
         </ul>
     </section>
+
+    <section class="works">
+      <div class="card">
+        <img src="images/snapshort_project1.png" alt="portfolio project" width="295"
+         height="220">
+         <h1 class="header-title">Tonic</h1>
```

Comment:
[OPTIONAL]
- [ ] It's better if the webpage has only one `h1` tag and the rest of the subheaders are `h2`, `h3`...